pkgs/development/tools/database/movine/default.nix

```
@@ -0,0 +1,48 @@
+{ rustPlatform, pkgs, fetchFromGitHub, lib, ... }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "movine";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "byronwasti";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0rms8np8zd23xzrd5avhp2q1ndhdc8f49lfwpff9h0slw4rnzfnj";
+  };
+
+  cargoSha256 = "sha256-4ghfenwmauR4Ft9n7dvBflwIMXPdFq1vh6FpIegHnZk=";
+
+  nativeBuildInputs = with pkgs; [ pkg-config ];
+  buildInputs = with pkgs; [ postgresql_12 sqlite ] ++ (
+    if !stdenv.isDarwin then [ openssl ] else
+      [ Security ]
+  );
+
+  meta = with lib; {
+    description = ''
+      A migration manager written in Rust that attempts to be
+      smart yet minimal
+    '';
```

```suggestion
    description = ''A migration manager written in Rust that attempts to be smart yet minimal'';
```
This is usually a oneliner

---

Should I just commit this suggestion with the GitHub-preset message "Update pkgs/development/tools/database/movine/default.nix", or what's the convention here?

---

make a new commit, but then [squash](https://gitbetter.substack.com/p/how-to-squash-git-commits) it into the first

---

Resolve the conversation, since you fixed this